components/Tabs/index.js

```diff
@@ -7,3 +7,18 @@
 //
 //  The tab component should look like this:
 //    <div class="tab">topic here</div>
+
+const entryTopics = document.querySelector('.topics');
+
+axios.get("https://lambda-times-backend.herokuapp.com/topics")
+    .then(response => {
+        response.data.topics.forEach(item => {
+            let tempVal = document.createElement('div');
```

Comment:
Nice job. This definitely shows that you understood the concepts to meet MVP without creating a separate component function.

Why did you choose not to make a component function?